tox.ini

```diff
@@ -50,7 +50,8 @@ commands =
 
 [testenv:flake8]
 recreate = True
-deps = flake8
+deps =
+    flake8==3.7.9  # 3.8.0 triggers "AttributeError: 'Namespace' object has no attribute 'output_file'"
```

Comment:
@mpenkov any idea what's going on?

Reply:
Sounds like a flake8 bug. Can we reproduce it with a minimal example? If so, we should tell the flake8 guys.